DxDiagram improve Expr descriptions (#620)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/edges/lineTypeExpr.md

The current edge's data object.

---

> The specified field or expression must return `orthogonal` or `straight` value.

Reviewer: ```suggestion
The specified field or expression should return `orthogonal` or `straight`.
```
